tests/main/osutil-disks/task.yaml

```diff
@@ -0,0 +1,21 @@
+summary: test osutil disk.Size()
+
+# ubuntu-core doesn't have go :-)
+# ppc64el disabled because of https://github.com/snapcore/snapd/issues/2503
```

Comment:
what is this issue by the way, we have this link elsewhere in the code but since issues on snapd were disabled I can't see what this originally was

Reply:
Yeah, I can't actually access this myself - I removed it for now, I think it was just cargo-culted so let's see.